scripts/security/phase4_enforcement_probe.py

```
@@ -0,0 +1,577 @@
+    runtime_conn.commit()
+
+    _set_tenant_context(runtime_conn, tenant_a)
+    visible_a = _count_marked_dead_events(runtime_conn, marker)
+    _set_tenant_context(runtime_conn, tenant_b)
+    visible_b = _count_marked_dead_events(runtime_conn, marker)
+    _set_tenant_context(runtime_conn, None)
+    missing_context_visible: int | None = None
+    missing_context_exception: str | None = None
+    try:
+        missing_context_visible = _count_marked_dead_events(runtime_conn, marker)
+    except Exception as exc:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)